### DISCUSSION TITLE
Bug With Data Provider filters

### DISCUSSION THREAD

**Discussion by swing18** [Budibase/budibase#6436] [General]
I've noticed a bug when defining a filter for Data Providers, that once you click save on your filter for the first time, changing it will have the desired effect on your design screen, but not in app preview. 

App preview will always use the first filter, and when you go back to look at the filter after launching app preview, it will have reverted back to the first filter.

The simple solution which I've found works is deleting the data provider and making a new one, making sure to get the filter right the first time. However I figured I'd post about it in case it could be helpful to someone else.

**Comment by melohagan** [comment]
Hi @swing18 

Thank you for bringing this to our attention. Would you be willing to raise a bug: https://github.com/Budibase/budibase/issues/new/choose

Please provide the steps to reproduce alongside some screenshots
